lighthouse-core/audits/main-thread-tasks.js

```diff
@@ -32,18 +32,16 @@ class MainThreadTasks extends Audit {
     const tasks = await MainThreadTasksComputed.request(trace, context);
 
     const results = tasks
-      .filter(task => task.duration > 5 && task.parent)
+      .filter(task => task.duration > 5 && !task.parent)
```

Comment:
maybe add a comment that top level === no parent here? Easy to miss the point of the filter though it's obvious once you realize :)

Reply:
yeah good call, I mean I missed it the first time around 😆